### PR TITLE
Update to v2.0 readme links

### PR DESCRIPTION
**Pull request by karnthis**
### What does this PR do?
Updated the v2.x readme to use v2.x docs links. Corrected minor typo.

### Motivation
I want to help others that are new to Traefik find the info they need to start with getting frustrated with out-of-date docs.

### More

- [ ] ~Added/updated tests~
- [x] Added/updated documentation

### Additional Notes

Keep the good work going!

### CODE REVIEW

**Comment by karnthis** [comment]
forgot to change branch selection before submit.

File: README.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ git clone https://github.com/containous/traefik
 
 ## Introductory Videos
 
+:warning: Please be aware that these videos are for v1.X. The old configurations for Traefik v1.X are NOT compatible with the v2.X config as of now. If you're testing out v2, please ensure you are using a [v2 configuration](https://docs.traefik.io/v2.0/).
```
Comment (dtomcej) [inline README.md]:
```suggestion
:warning: Please be aware that these videos are for v1.X. The old configurations for Traefik v1.X are NOT compatible with Traefik v2. If you're testing out v2, please ensure you are using a [v2 configuration](https://docs.traefik.io/v2.0/).
```

**Comment by karnthis** [comment]
Done - requested change made

**Comment by karnthis** [comment]
Respectfully, these changes are to the v2 branch and its readme. Why should v2 direct to useless documentation?

**Comment by ldez** [comment]
Because we keep the readme on `master` and `v2.0` branches synchronized.

**Comment by karnthis** [comment]
Ah I see. In that case should I change to offering links to both latest and v2 documentation? Something as simple as "for v2 alpha documentation click here"? Currently you can only find the alpha docs if you know where to look for them.

**Comment by ldez** [comment]
in https://docs.traefik.io/ you got a menu with all Traefik version.

**Comment by karnthis** [comment]
I know about the drop-down, that is part of my point. It is a poor user experience to view a readme, be directed to the documentation resource, see the documentation, and then because you didn't know to select a small option on the side that should have been already been selected since you came there looking for **that** information, you become frustrated because nothing is working as you are instructed to do.

The point I am trying to make is _something_ in that workflow _somewhere_ needs to indicate that the user was **not** taken to the v2 docs as they might have expected.

**Comment by ldez** [comment]
You can add sentence in the documentation section: https://github.com/containous/traefik/tree/v2.0#documentation

**Comment by karnthis** [comment]
Rolled back requested changes, added line directing v2 users to v2 docs

**Pull request merged**